From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Todd Kjos <tkjos@google.com>
Date: Thu, 11 Jun 2020 13:53:46 -0700
Subject: Revert "ANDROID: gki_defconfig: disable KPROBES"

This reverts commit 953d8eab92876a323f7f76b2dd19e0d78eea6add.

Turn KPROBES back on now that selinux protection is in place

Bug: 149659981
Change-Id: I22462081fca1af9758e57bcd75c88bdd29d23f8b
Signed-off-by: Todd Kjos <tkjos@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -79,6 +79,7 @@ CONFIG_ARM_SCPI_PROTOCOL=y
 # CONFIG_EFI_ARMSTUB_DTB_LOADER is not set
 CONFIG_CRYPTO_SHA2_ARM64_CE=y
 CONFIG_CRYPTO_AES_ARM64_CE_BLK=y
+CONFIG_KPROBES=y
 CONFIG_SHADOW_CALL_STACK=y
 CONFIG_LTO_CLANG=y
 CONFIG_CFI_CLANG=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -57,6 +57,7 @@ CONFIG_CPU_FREQ_TIMES=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_IA32_EMULATION=y
+CONFIG_KPROBES=y
 CONFIG_LTO_CLANG=y
 CONFIG_CFI_CLANG=y
 CONFIG_MODULES=y
